pkgs/by-name/in/intel-compute-runtime/package.nix

```diff
@@ -39,6 +39,7 @@ stdenv.mkDerivation rec {
     (lib.cmakeFeature "OCL_ICD_VENDORDIR" "${placeholder "out"}/etc/OpenCL/vendors")
     # The install script assumes this path is relative to CMAKE_INSTALL_PREFIX
     (lib.cmakeFeature "CMAKE_INSTALL_LIBDIR" "lib")
+    (lib.cmakeBool "NEO_DISABLE_MITIGATIONS" true)
```

Comment:
Can we add a comment with https://bugs.launchpad.net/ubuntu/+source/intel-compute-runtime/+bug/2110131 ?